src/client/cosmos.ts

```diff
@@ -60,9 +65,9 @@ export default class CosmosDbClient {
       };
       const client = new CosmosClient({
         endpoint: account,
-        key,
         connectionPolicy,
-        consistencyLevel: 'Session'
+        consistencyLevel: 'Session',
+        ...connectionInfo
```

Comment:
this is a bit too clever. would recommend making the code more obvious as to the auth variations you can have